src/link/Wasm.zig

```diff
@@ -2836,14 +2836,44 @@ pub const Feature = packed struct(u8) {
         @"shared-mem",
 
         pub fn fromCpuFeature(feature: std.Target.wasm.Feature) Tag {
-            return @enumFromInt(@intFromEnum(feature));
+            return switch (feature) {
```

Comment:
Good move, this looks a lot better. Sorry about the trouble, I should have done this from the beginning.